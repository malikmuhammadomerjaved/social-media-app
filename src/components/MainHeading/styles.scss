@import "../../styles/commons.scss";

.main-heading-wrapper {
  @include flexView(center, unset, column);

  @media (min-width: $tablet-screen) {
    @include flexView(flex-end, unset, column);
  }

  .plain-heading {
    font-weight: 700;
    color: $primary-color;

    &.gradient {
      background: -webkit-linear-gradient(
        90deg,
        $gradient-from 24.2%,
        $gradient-to 120.46%
      );
      -webkit-background-clip: text;
      -webkit-text-fill-color: transparent;
    }
  }
}
