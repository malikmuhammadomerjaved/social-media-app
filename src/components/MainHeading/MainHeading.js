import "./styles.scss";

const MainHeading = () => {
  return (
    <article className="main-heading-wrapper">
      <h1 className="plain-heading">Imagine if</h1>
      <h1 className="plain-heading gradient">Snapchat</h1>
      <h1 className="plain-heading">had events.</h1>
    </article>
  );
};

export default MainHeading;
