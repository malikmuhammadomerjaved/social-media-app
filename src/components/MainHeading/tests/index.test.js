import { render } from "@testing-library/react";
import MainHeading from "../MainHeading";

describe("<MainHeading />", () => {
  const { container } = render(<MainHeading />);

  it("should match MainHeading component to snapshot", () => {
    expect(container).toMatchSnapshot();
  });
});
