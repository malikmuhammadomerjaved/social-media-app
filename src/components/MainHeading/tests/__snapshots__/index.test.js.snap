// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MainHeading /> should match MainHeading component to snapshot 1`] = `
<div>
  <article
    class="main-heading-wrapper"
  >
    <h1
      class="plain-heading"
    >
      Imagine if
    </h1>
    <h1
      class="plain-heading gradient"
    >
      Snapchat
    </h1>
    <h1
      class="plain-heading"
    >
      had events.
    </h1>
  </article>
</div>
`;
